src/main/java/com/conveyal/r5/transit/TransportNetwork.java

```diff
@@ -37,55 +38,60 @@
     public TransitLayer transitLayer;
 
     /**
-     * Cache a grid point set that covers the extents of this transport network. The PointSet itself caches linkages to
-     * street networks.
-     *
-     * Eventually we should save this point set and linkage but it's a little tricky since there's a linkage cache with
-     * references to streetlayers, a spatial index, etc. in there.
+     * A grid point set that covers the full extent of this transport network. The PointSet itself then caches linkages
+     * to street networks (the baseline street network, or ones with various scenarios applied). If they have been
+     * created, this point set and its linkage to the street network are serialized along with the network, which makes
+     * startup much faster. Note that there's a linkage cache with references to streetlayers in this GridPointSet,
```

Comment:
The WebMercatorGridPointset does not itself contain any references to a StreetLayer. Even if it did, it would be a reference to the network's default street layer, which is already being serialized. The serializer would spot the repeat reference and not serialize the streetLayer twice. The Guava loadingCache of linkages will not serialize its contents by design. That's why the TransportNetwork also has a linkedGridPointSet field. This means we're completely bypassing the cache for gridded analysis, but we've already essentially got two different methods, one gridded one not. Come to think of it, it would be more normalized to just store the linked grid pointset, and access the basic unlinked grid pointset via the linked pointset field.